From 6bb37d22af95d11df7f487f3e34abc97fa5fa9bf Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 28 Oct 2014 20:36:32 +0100
Subject: [PATCH] NEWS: well, it's Options= now, not Discard=

(cherry picked from commit c4ac990007cd0069bb7e76ec15dd731320f382fd)
---
 NEWS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/NEWS b/NEWS
index c85605f1d6..bf8d0ac242 100644
--- a/NEWS
+++ b/NEWS
@@ -59,7 +59,7 @@ CHANGES WITH 217:
           rotating media anymore, and nobody stepped up to actively
           maintain this component of systemd it has now been removed.
 
-        * Swap units can use Discard= to specify discard options.
+        * Swap units can use Options= to specify discard options.
           Discard options specified for swaps in /etc/fstab are now
           respected.
 
